_includes/header.html

```diff
@@ -1,4 +1,25 @@
 <header id="header" class="header" role="banner">
+    
+    <div class="block block__flush-top block__flush-bottom block__padded-bottom">
+        <div class="m-global-banner">
+            <div class="wrapper
+                        wrapper__match-content">
+            
+                <div class="m-notification m-notification__warning m-notification__visible">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 1200" class="cf-icon-svg"><path d="M500 105.2c-276.1 0-500 223.9-500 500s223.9 500 500 500 500-223.9 500-500-223.9-500-500-500zm-49.7 234.6c0-27.6 22.4-50 50-50s50 22.4 50 50v328.6c0 27.6-22.4 50-50 50s-50-22.4-50-50V339.8zm50 582.5c-39.6 0-71.7-32.1-71.7-71.7s32.1-71.7 71.7-71.7S572 811 572 850.6s-32.1 71.7-71.7 71.7z"></path></svg>
+                    <div class="m-notification_content">
+                        <div class="h4 m-notification_message">This site is deprecated.</div>
+                            <p class="m-notification_explanation">
+                                This site will no longer be updated and future updates will happen in
```

Comment:
My gut prefers "happen **at**" over "happen **in**" but I have no idea which is grammatically correct.